tests/integration/conftest.py

```diff
@@ -423,3 +426,9 @@ async def related_prometheus(ops_test: OpsTest, cos_model, _cos_lite_installed):
     with ops_test.model_context("cos") as model:
         log.info("Removing Prometheus Offer...")
         await model.remove_offer(f"{model.name}.prometheus-receive-remote-write", force=True)
+
+
+@pytest.fixture(scope="module")
+def timeout(request):
+    """Fixture to set the timeout for certain tests"""
+    return request.config.option.timeout
```

Comment:
some new lint rules after a branch update have gone into effect and need an easy patch up here